lib/job-simulation.rb

```diff
@@ -1,31 +1,50 @@
-require './Stack.rb'
-require './Queue.rb'
+require './lib/Stack.rb'
+require './lib/Queue.rb'
 
 class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
+    @waiting = Queue.new
+    job_seekers.times {|n|
+      @waiting.enqueue("Worker ##{n+1}")
+    }
 
+    @workers = Stack.new
+    jobs_available.times {
+      @workers.push(@waiting.dequeue)
+    }
```

Comment:
In ruby, you should use `do` `end` instead of the curly braces, unless it's a one-liner. I do love my curly braces though.